### PR TITLE
Removed English comments from translations

### LINKED ISSUE #2510: Remove English commented out code from translations
It seems it's common practice. We need another way for translators to follow the English files and know up to where they have translated. Maybe a simple HTML comment with the hash and then they can compare after that and see the diff.

Right now, many translations follow this practice which results in ~2x lines. All this does not come for free and it's bloating the codebase IMO.

Also, there's #2450.

### PR DESCRIPTION
**Pull request by grodd**
fixes #2510

### CODE REVIEW

**Comment by XhmikosR** [comment]
Without having another solution for translators, this shouldn't land.

Also, note there are test failures (extra blank lines).

File: locale/ja/docs/guides/buffer-constructor-deprecation.md [hunk 1]
```diff
@@ -13,19 +13,11 @@ layout: docs.hbs
 The Buffer() and new Buffer() constructors are not recommended for use due to security and usability concerns. Please use the new Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() construction methods instead.
 </div>
 
-* [Variant 1: Node.js ≤ 4.4.x および 5.0.0 — 5.9.x のサポートを終了](#variant-1) (*推奨*)
-* [Variant 2: ポリフィルの使用](#variant-2)
-* [Variant 3: セーフガード付きの手動検出](#variant-3)
+- [Variant 1: Node.js ≤ 4.4.x および 5.0.0 — 5.9.x のサポートを終了](#variant-1) (*推奨*)
```
Comment (XhmikosR) [inline locale/ja/docs/guides/buffer-constructor-deprecation.md:16]: This kind of changes shouldn't be in this patch.

**Comment by MaledongGit** [comment]
@grodd, thanks for your contribution!

However, there must be something wrong with your submit, so you should run `npm run test` locally before your submittion.

**Review by ghost — CHANGES_REQUESTED** [review]
Hello,

First I'd say welcome your contributions!

Before your submition, please have a review first and run `npm run test`.

Then please DO NOT remove the comment in Korean and Japanese, because their authors want to keep them as a history track which they can compare and do translations, it's their habits (Waiting for their response).

If you have any other changes, please repush them and we'll have a review of them.

**Comment by XhmikosR** [comment]
This has become outdated already.

I lean towards closing this until we have a proper solution to track translations.

**Pull request closed without merging**